modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -7357,6 +7357,8 @@ feature.flag.LPD-16469.description=Users have the ability to bulk manage permiss
 feature.flag.LPD-16469.title=Bulk Manage Permissions for Web Content Articles
 feature.flag.LPD-21265.description=Users can set default permissions for new web content (pages, blog posts, etc.) during the creation process.
 feature.flag.LPD-21265.title=Default Permissions
+feature.flag.LPD-21636.description=Users can define and manage a Picklist to configure Product Specification value.
+feature.flag.LPD-21636.title=Configure Specification value with Picklist
```

Comment:
value should not be values? (in both the sentences)

Reply:
actually the value for the specification is only one

Reply:
@JamesAGarcia can you please have a look at this? Thank you!